File: back/KaemeWebApi/Dominio/Models/Cliente.cs
namespace Dominio.Models 
{

    public partial class Cliente
    {
        public int? Id { get; set; }
        public string Nome { get; set; }
        public DateTime? Data_Nasc { get; set; }
        public string? Signo { get; set; }
        public string? CPF { get; set; }
        public string? Telefone { get; set; }
        public string? Email { get; set; }
        public string? Instagram { get; set; }
        public string? Meio_Captacao { get; set; }
        public string? CEP { get; set; }
        public string? Logradouro { get; set; }
        public int? Numero { get; set; }
        public string? Complemento { get; set; }
        public string? Bairro { get; set; }
        public string? Cidade { get; set; }
        public string? Uf { get; set; }
        public char Sexo { get; set; }
        public DateTime Data_Criacao { get; set; }
        public int Criado_Por { get; set; }
    }

}

File: front/kaemeweb/src/app/modules/shared/models/cliente.ts
export interface Cliente {
  id?: number;
  nome: string;
  data_Nasc?: Date | null;
  signo?: string | null;
  cpf?: string | null;
  telefone?: string | null;
  email?: string | null;
  instagram?: string | null;
  meio_Captacao?: string | null;
  cep?: string | null;
  logradouro?: string | null;
  numero?: number | null;
  complemento?: string | null;
  bairro?: string | null;
  cidade?: string | null;
  uf?: number | null;
  sexo: string | null;
  data_Criacao: Date;
  criado_Por: number;
}
